hydra-node/integration/IntegrationSpec.hs

```diff
@@ -173,10 +205,14 @@ simulatedChainAndNetwork = do
   broadcast nodes msg = readTVarIO nodes >>= mapM_ (`handleMessage` msg)
 
 startHydraNode :: Natural -> (HydraNode MockTx IO -> IO Connections) -> IO (HydraProcess IO MockTx)
-startHydraNode nodeId connectToChain = do
+startHydraNode nodeId connectToChain =
+  startHydraNodeTraced nodeId connectToChain nullTracer
+
+startHydraNodeTraced :: Natural -> (HydraNode MockTx IO -> IO Connections) -> Tracer IO (HydraNodeLog MockTx) -> IO (HydraProcess IO MockTx)
```

Comment:
Could always return a `TVar` with something like this signature `startHydraNodeTraced :: Natural -> (HydraNode MockTx IO -> IO Connections) -> Tracer IO (HydraNodeLog MockTx) -> IO (HydraProcess IO MockTx, TVar [HydraNodeLog])`